modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -738,80 +745,78 @@ WHERE provider = 'oidc1'
 
 ======
 
-[NOTE]
-====
-The `SHOW USER name PRIVILEGES` command is described in xref:authentication-authorization/manage-privileges.adoc#access-control-list-privileges[Listing privileges].
-====
-
-
 [[access-control-create-users]]
 == Creating users
 
-Users can be created using `CREATE USER`.
+You can create users using one of the following Cypher commands, depending on whether you want to create a new user or replace an existing one.
+In both cases, you can specify the user's password, whether they must change it at the next login, their status, home database, and auth provider settings.
+The `SET` clauses can be applied in any order.
+It is mandatory to specify a `SET PASSWORD` or a `SET AUTH` clause or both because users must have at least one auth provider.
```

Comment:
```suggestion
It is mandatory to specify a `SET PASSWORD` and/or at least one `SET AUTH` clause because users must have at least one auth provider.
```